kuma/users/views.py

```diff
@@ -467,14 +470,54 @@ def user_edit(request, username):
 
             return redirect(edit_user)
 
+    subscription_info = None
+    stripe_customer = get_stripe_customer(edit_user)
+    # XXX Might be nice to explain WHY we're doublechecking the user email.
+    if stripe_customer and stripe_customer.email == edit_user.email:
```

Comment:
can we extract that into a separate function again? I find user_edit to be already a bit much, readability wise.

Reply:
Actually. I'm just going to kill this. Mostly because we no longer have a record of WHY we're doing it and the simple above-mentioned use case where a MDN user changes her primary email and suddenly it appears as if she doesn't have a subscription.